src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1118,7 +1113,7 @@ public static bool IsParameterTypeContext(this SyntaxTree syntaxTree, int positi
                 tokenOnLeftOfPosition = syntaxTree.FindTokenOnLeftOfPosition(position, cancellationToken);
             }
 
-            if (syntaxTree.IsParameterModifierContext(position, tokenOnLeftOfPosition, cancellationToken))
+            if (syntaxTree.IsParameterModifierContext(position, tokenOnLeftOfPosition, includeOperators: true, out _, out _))
```

Comment:
are params allowed in operators?  consider making false.  (but i don't really care).

Reply:
This is inside `IsParameterTypeContext`, not `IsParamsModifierContext`. Parameter types are allowed in operators.

Reply:
ah gotcha, i missed the collapsed section.